mlflow/models/evaluation/default_evaluator.py

```diff
@@ -1058,19 +1058,16 @@ def _generate_model_predictions(self):
         Helper method for generating model predictions
         """
         if self.model_type == "classifier":
-            self.label_list = np.unique(self.y)
+            self.label_list = sorted(np.unique(self.y))
```

Comment:
Do we need `sorted`?

https://numpy.org/doc/stable/reference/generated/numpy.unique.html

> Returns the **sorted** unique elements of an array. There are three optional outputs in addition to the unique elements:

Reply:
Yes, because we assume that the first label is negative and the second is positive in the logger statement below. If pos_label is not contained in the set, then the convention should be that the lower value is negative.